src/main/java/at/roteskreuz/covidapp/config/SecurityConfig.java

```diff
@@ -31,7 +31,7 @@ protected void configure(HttpSecurity http) throws Exception {
 				.csrf().disable()
 				.authorizeRequests()
 				.antMatchers("/api/v" + appVersion + "/*").permitAll()
-				//.antMatchers("/swagger*").permitAll()
+				.antMatchers("/swagger*").permitAll()
```

Comment:
Was the comment intentionally removed or was this an unwanted side effect of the style fix?

Reply:
oh thanks, oversee that, tested some other stuff and did commits between that....